db/migrate/20190705060630_add_cloned_value_to_parameters.rb

```diff
@@ -0,0 +1,11 @@
+class AddClonedValueToParameters < ActiveRecord::Migration[5.2]
+  def change
+    add_column :parameters, :cloned_value, :text
+
+    if column_exists?(:parameters, :cloned_value)
+      Parameter.unscoped.find_each do |param|
+        param.save
```

Comment:
This would trigger all callbacks and register audits as well, perhaps `param.update_columns(cloned_value: param.set_cloned_value)`?

Reply:
@tbrisker, I have updated PR with suggested changes.